compiler/luci/pass/src/ResolveCustomOpAddPass.test.cpp

```diff
@@ -34,3 +150,69 @@ TEST_F(ResolveCustomOpAddPassTest, name)
   auto const name = _pass.name();
   ASSERT_NE(nullptr, name);
 }
+
+TEST_F(ResolveCustomOpAddPassTest, simple_test_CustomBroadcastTo)
+{
+  _g.init();
+  _g.addV2()->inputs(1, _g.custom_broadcastTo_out());
+
+  auto ret = _pass.run(_g.g());
+  EXPECT_EQ(true, ret);
+
+  auto add = dynamic_cast<luci::CircleAdd *>(_g.output(0)->from());
+  EXPECT_NE(nullptr, add);
+
+  auto broadcastTo_input = dynamic_cast<const luci::CircleConst *>(add->y());
+  EXPECT_NE(nullptr, broadcastTo_input);
+  EXPECT_EQ(1, broadcastTo_input->at<loco::DataType::FLOAT32>(0));
+  EXPECT_EQ(2, broadcastTo_input->at<loco::DataType::FLOAT32>(1));
+  EXPECT_EQ(3, broadcastTo_input->at<loco::DataType::FLOAT32>(2));
+
+  auto input = dynamic_cast<luci::CircleNode *>(add->x());
+  EXPECT_NE(nullptr, input);
+  EXPECT_EQ(2, input->dim(0));
+  EXPECT_EQ(3, input->dim(1));
+}
+
+TEST_F(ResolveCustomOpAddPassTest, simple_test_BuiltinBroadcastTo)
+{
+  _g.init();
+  _g.addV2()->inputs(1, _g.builtin_broadcastTo());
+
+  auto ret = _pass.run(_g.g());
+  EXPECT_EQ(true, ret);
+
+  auto add = dynamic_cast<luci::CircleAdd *>(_g.output(0)->from());
+  EXPECT_NE(nullptr, add);
+
+  auto broadcastTo_input = dynamic_cast<luci::CircleConst *>(add->y());
```

Comment:
ditto

Reply:
@seanshpark 
Thank you for your comments!
I modified `input` to `input_0` and `broadcastTo_input` to `input_1` so as not to confuse. :)